pants

```diff
@@ -2,6 +2,39 @@
 # Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
+REPO_ROOT=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd -P)
+
+_interpreter_version_sentinel_file_path='.pants.d/.pants-python-version'
```

Comment:
I recommend changing to `.python-interpreter-constraints`

Also because it's a local variable, I think you can remove the `_` here and elsewhere. That's personal style preference though.

```suggestion
interpreter_constraints_sentinel_file_path='.pants.d/.python-interpreter-constraints'
```

Reply:
Removed `_` for consistency (thanks for pointing that out, will stick with lowercase in this repo!) and changed to `.python-interpreter-constraints`!